src/test/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringdataplatformapi/controller/LocationControllerTest.kt

```diff
@@ -33,10 +36,17 @@ class LocationControllerTest {
     Assertions.assertThat(result.statusCode).isEqualTo(HttpStatus.OK)
   }
 
+  fun confirmNoErrorForAggregation(
+    result: ResponseEntity<LocationAggregationResponse>,
+    expected: ResponseEntity<LocationAggregationResponse>,
+  ) {
+    Assertions.assertThat(result.body?.error).isEqualTo("")
+    Assertions.assertThat(result.body?.message).isEqualTo(expected.body?.message)
+    Assertions.assertThat(result.statusCode).isEqualTo(HttpStatus.OK)
+  }
+
   @Test
   fun `getAllLocations should return No data found when no location exists`() {
-    Mockito.`when`(locationService.getAllLocations()).thenReturn(listOf())
```

Comment:
Why is this removed? Does this no longer get called by new-line 51? :
`val result = LocationController(locationService).getAllLocations()`